crates/warg/src/package/validate.rs

```diff
@@ -71,62 +70,76 @@ pub enum ValidationError {
     TimestampLowerThanPrevious,
 }
 
-#[allow(clippy::large_enum_variant)]
-#[derive(Default, Debug, Clone, PartialEq, Eq)]
-pub enum ValidationState {
-    #[default]
-    Uninitialized,
-    Initialized(ValidationStateInit),
+/// Represents information about a package yanking.
+#[derive(Debug, Clone, PartialEq, Eq, Serialize, Deserialize)]
+pub struct Yank {
```

Comment:
I think since the entry for yanking is also just called `Yank` (albeit with enum prefix), this should maybe be called `YankInfo` or `YankContext`.